content/docs/hosting/BYOC.md

```diff
@@ -98,169 +99,178 @@ With Appbase.io Dashboard you can easily share cluster with your team members. E
+**Step 1-** Get TLS certificate with `.pem` & `.key` file
 
-#### **Getting Arc URL**
+> **Note:** You can obtain certificate using [https://letsencrypt.org/](https://letsencrypt.org/) or any other provider.
 
-Once the EC2 machine is created successfully, you can get Public DNS / IP address from the EC2 dashboard. This can be used with `USERNAME` and `PASSWORD` to access ElasticSearch data using Arc.
+**Step 2-** Access your EC2 instance using `ssh`. From EC2 dashboard you can select instance and click on **Connect** to get `ssh` login details about your machine.
 
-![](https://i.imgur.com/PDs8DK0.png)
+![EC2 Connect Screenshot](https://i.imgur.com/NRW1P6c.png)
 
-**Example**
+**Step 3-** Copy `.pem` file content to `/etc/pki/nginx/server.crt` using `vim`/`nano` editor
 
 ```bash
-curl -u="USERNAME:PASSWORD" http://EC2-PUBLIC-[IP/DNS]
+# Open the file
+sudo vim /etc/pki/nginx/server.crt
+# past the code
+cmd/ctrl + v
+# save the file
+:wq
 ```
 
-You can also point your domain to EC2 instance in order to access data using your domain. For more information on how to register/point domain to EC2 instance you can check [docs](https://aws.amazon.com/getting-started/tutorials/get-a-domain/).
+**Step 4-** Copy `.key` file content to`/etc/pki/nginx/private/server.key` using similar approach mentioned above.
 
-#### **Accessing Arc Dashboard**
+**Step 5-** Open `sudo vim /etc/nginx/nginx.conf`
 
-Alright now in order to access all the Arc Services, let us Sign in to [Arc dashboard](https://arc-dashboard.appbase.io/login) and start accessing data from there.
+**Step 6-** Configuration for TLS is already there in `nginx.conf`, you just need to uncomment it.
 
--   **Step 1 -** Sign in into [Arc dashboard](https://arc-dashboard.appbase.io/login) using your EC2 Public Domains / Domain with Username and Password set in the ENV
+-   Uncomment line `125` - `213`
+-   Comment out or remove server listening for PORT 80.
 
-![](https://i.imgur.com/qVSHx0F.png)
+**Step 7-** Restart nginx service using following command
 
-And we are ready with our Arc Setup using AMI from AWS marketplace.
+```bash
+sudo systemctl restart nginx
+```
 
-#### **Changing Environment Variables**
+#### **Configuring Arc**
 
-In order to update any environment variable here, you need to do `ssh` login into your EC2 machine. You can connect with EC2 machine via `ssh` using `.pem` key file that you must created / used while configure and creating and EC2 using Arc AMI.
+Once EC2 machine is up and running you can open EC2 public DNS OR if EC2 DNS is pointing to your custom domain, then you can open that URL in browser. You might come across following page initially.
 
-You can click on **Connect** button from EC2 Dashboard and see the instructions to do ssh connection.
+![](https://i.imgur.com/huRzI3e.png)
 
-![](https://i.imgur.com/cS8LKkF.png)
+You can start configuring Arc by clicking on _configure page_ link OR opening `/configure` URI. Below are steps that you can follow
```

Comment:
You can start configuring Arc by clicking on _configure page_ link OR opening `/configure` URI. You can follow the below steps: